### DISCUSSION TITLE
When used with company-tabnine, can get following failing menu, i even can't quit it with Ctrl+G

### DISCUSSION THREAD

**Discussion by zw963** [company-mode/company-mode#1300] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/549126/149730641-a290d678-8dbc-4f24-9350-05737653cfb1.png)

as you can see, my cursor is outside company candidate menu, but that menu still not disappear.

because i use company-posframe, i guess it caused by this plugin, i will remove it, and try several days, to check if this issue still happen.

**Comment by dgutov** [comment, marked as answer]
Try updating to the latest `posframe` and `company-posframe`.

But if the problem persists, you'll need to report that issue there.
